server/app/import.go

```diff
@@ -287,16 +306,28 @@ func (a *App) ImportBoardJSONL(r io.Reader, opt model.ImportArchiveOptions) (*mo
 				Synthetic:       boardMember.Synthetic,
 			}
 			if _, err2 := a.AddMemberToBoard(bm); err2 != nil {
-				return nil, fmt.Errorf("cannot add member to board: %w", err2)
+				return fmt.Errorf("cannot add member to board: %w", err2)
 			}
 		}
 	}
+	return nil
+}
 
-	// find new board id
-	for _, board := range boardsAndBlocks.Boards {
-		return board, nil
+func validateAttachmentIfPresent(block *model.Block) error {
+	if block.Type == model.TypeImage || block.Type == model.TypeAttachment {
```

Comment:
Same as before, we should validate regardless of `block.Type`, and use constants for `model.FileId` and `model.AttachmentId`